funnel/templates/search.html.jinja2

```diff
@@ -8,9 +12,88 @@
+        <div class="user__box">
+          {{ useravatar(item.obj.user) }}
+          <div class="user__box__header comment__header__details__user">
+            <h4 class="mui--text-bold zero-top-margin zero-bottom-margin"><a href="{{ item.url }}">{{ item.title_html }}</a></h4>
+            <h3 class="mui--text-body2 zero-top-margin zero-bottom-margin">
+              {%- if item.obj.first_user.fullname %}<span>{{ item.obj.first_user.fullname }}</span> <span class="mui--text-light">in</span>{% endif %} <span>{{ item.obj.project.full_title }} </span>
+              {%- if item.obj.session and item.obj.session.start_at %}
+                <span><span class="mui--text-light">at</span> {{ item.obj.session.start_at_localized|time }}–{{ item.obj.session.end_at_localized|time }} <span class="mui--text-light">on</span> {{ item.obj.session.start_at_localized|date }}</span>
+              {% endif %}
+            </h3>
+          </div>
+        </div>
+        <p class="mui--text-body2 mui--text-light snippets"><faicon icon='search' css_class="search-icon" baseline=false></faicon> {{ item.snippet_html }}</p>
+      {%- elif type == 'update' %}
+        <div class="user__box">
+          {{ useravatar(item.obj.user) }}
+          <div class="user__box__header comment__header__details__user">
+            <h4 class="mui--text-bold zero-top-margin zero-bottom-margin"><a href="{{ item.url }}">{{ item.title_html }}</a></h4>
+            <h3 class="mui--text-body2 zero-top-margin zero-bottom-margin">
+              {%- if item.obj.user.fullname %}<span>{{ item.obj.user.fullname }}</span> <span class="mui--text-light">in</span>{% endif %} <span>{{ item.obj.project.full_title }} </span>
+            </h3>
+          </div>
+        </div>
+        <p class="mui--text-body2 mui--text-light snippets"><faicon icon='search' css_class="search-icon" baseline=false></faicon> {{ item.snippet_html }}</p>
+      {%- elif type == 'comment' %}
+        <div class="user__box">
+          {{ useravatar(item.obj.user) }}
+          <div class="user__box__header comment__header__details__user">
+            <h4 class="mui--text-bold zero-top-margin zero-bottom-margin"><a href="{{ item.url }}">{{ item.obj.title }}</a></h4>
+            <p class="mui--text-body2 mui--text-light zero-bottom-margin">{{ item.obj.created_at|date }} {%- if item.obj.edited_at %} {% trans date=item.obj.edited_at|date %}edited {{ date }}{% endtrans %}{% endif %}</p>
+          </div>
+        </div>
+        <p class="mui--text-body2 mui--text-light zero-top-margin snippets"><faicon icon='search' css_class="search-icon" baseline=false></faicon> {{ item.snippet_html }}</p>
+      {% endif %}
+    </div>
+  {% endfor %}
+  {% if results['has_next'] %}
+    <div lass="tab-content__results user grid__col-sm-12 {%- if type == 'project' or type == 'profile' %} grid__col-sm-6 grid__col-lg-4{% endif %}" hx-get="{{ url_for('SearchView_search') }}?q={{ query }}&type={{ type }}&page={{ results['next_num'] }}"
```

Comment:
These need URL escaping.

Reply:
To get Flask to do this for you:

`{{ url_for('SearchView_search',  q=query, type=type, page=results['next_num']) }}`